### PR TITLE
Updates for Smallpeice 2024

### PR DESCRIPTION
**Pull request by WillB97**
- Simulator pages
- Pi LEDs
- Updated vision docs
- General corrections

### CODE REVIEW

File: docs/programming/vision/index.md [hunk 1]
```diff
@@ -4,104 +4,171 @@
...
 
-You can download the markers from the [resources page](../../resources).
+For information on markers, see the [markers page](./markers).
 
-## Searching for markers
+## Camera
 
-Assuming you have a webcam connected, you can use `robot.camera.see()` to take a picture. The software will process the picture
-and returns a list of the markers it sees.
+The interface to the vision system is through the camera, accessible through `robot.camera`.
```
Comment (sedders123) [inline docs/programming/vision/index.md:18]: Worth mentioning somewhere here that they can see an annotated version of the last image taken in the Web Interface?

File: docs/simulator/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Simulator
+
+The simulator is a useful development tool that allows you to become familiar with our API and test out robotics concepts even if you haven't finished building your robot yet.
+
+The simulator is built around the [Webots](https://cyberbotics.com/#download) platform, which runs the simulation.
+You control the virtual robot using the [sbot API](../programming/), just like on the physical robots.
+
+## Setup
+
+To use the simulator, you will need to install the simulator software and download our simulation environment.
```
Comment (sedders123) [inline docs/simulator/index.md:10]: Should we mention that they don't need to do this on labs machines/our laptops?

File: docs/simulator/setup.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 4.1
+---
+
+# Setting up the Simulator
+
+## Required Software
+
+In order to use the simulator a few set-up steps need to be done.
+First you need to install Python 3.9+ and Webots R2023b.
```
Comment (sedders123) [inline docs/simulator/setup.md:10]: Should this be the sane Python version we use on the kit 3.11?

**Review by sedders123 — COMMENTED** [review]
Looks good thanks! Left a few thoughts inline

**Comment by prophile** [comment]
Slightly weirdly wrapping text:

Tip showing the phrase 'Open with' split over a very long vertical space

**Review by prophile — COMMENTED** [review]
"Smallpeice CER" in the title isn't defined, the course title is "Computing, Robotics & Electronics (Summer School)" which would be CRE or CRESS.

File: docs/simulator/usage.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+### The Console
+
+The console at the bottom of the screen shows all the logs produced by Webots which includes logs from your code.
+
+Where your code logs are are printed they are prefixed with the zone number and simulation time.
+Lines are displayed in red if they were printed to standard error instead of standard output.
+
+There are also some Webots error messages that you may see, such as:
+
+**WARNING: The current physics step could not be computed correctly.**
```
[the remark this answers is not in the document]
Comment (WillB97) [reply docs/simulator/usage.md:75]: That's what webots prints.

File: docs/simulator/usage.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+### The Camera Overlay
+
+The camera overlay shows the images from the robot's camera.
+This only updates when `robot.camera.see()` is called in your code.
+
+This image is the raw image that the robot sees, and is not processed in any way.
+To see the processed image, look at the [saving images](../programming/vision/#saving-camera-output) section of the camera interface to see how to save annotated images.
+
+If the camera overlay is closed and you want to get it back, you can use the [Reopening the Camera Overlay](#reopening-the-camera-overlay) instructions.
```
[the remark this answers is not in the document]
Comment (WillB97) [reply docs/simulator/usage.md:45]: This is more clear about where the link is going.

**Pull request merged**